display.go

```diff
@@ -83,6 +94,109 @@ func updateStaticContents() {
 	updateLabelIfChanged("ui_Status_Content_Device_Id_Content_Label", GetDeviceID())
 }
 
+// setDisplayBrightness sets /sys/class/backlight/backlight/brightness to alter
+// the backlight brightness of the JetKVM hardware's display.
+func setDisplayBrightness(brightness int) error {
+	if brightness > 100 || brightness < 0 {
```

Comment:
This can actually go all the way up to 255... although at that level, it is really bright.

Reply:
I was going to say I went with 100 because that's the default, but it's actually not, they've got it set to 64 out of the box.
Though I'm glad you brought that up because I had considered writing it as a percentage from 0-100, but since it goes to 255, that isn't right!

Reply:
That's what things like the edid selector do... its all frontend stuff to make it happen. I don't really see needing to go above 100, and there may be power limits that it would hit, especially as the extension port starts to be used. Would need input from the hardware team on that one. To be safe, should probably just limit it to 64.

Reply:
I'll limit it to 64 in the frontend and keep the 100 limit on this function I think. @adamshiervani / Lian your input on this would be appreciated when you get time.

Reply:
100 is a pretty good choice as max. the screen get warm at max brightness of 255